File: Java/Solution_28.java
public class Solution_28 {
    public int strStr(String haystack, String needle) {
        if (needle.isEmpty()) {
            return 0;
        }
        return haystack.indexOf(needle);
    }

    public static void main(String[] args) {
        Solution_28 solution = new Solution_28();

        String haystack1 = "sadbutsad";
        String needle1 = "sad";
        String haystack2 = "leetcode";
        String needle2 = "leeto";

        System.out.println(solution.strStr(haystack1, needle1));  // Output: 0
        System.out.println(solution.strStr(haystack2, needle2));  // Output: -1

    }
}

File: Python/Solution_28.py
class Solution_28(object):
    def strStr(self, haystack, needle):
        """
        :type haystack: str
        :type needle: str
        :rtype: int
        """
        return haystack.find(needle)
    
if __name__ == "__main__":
    solution = Solution_28()
    
    haystack1 = "sadbutsad"
    needle1 = "sad"
    haystack2 = "leetcode"
    needle2 = "leeto"

    print("Converted Zigzag Output:", solution.strStr(haystack1, needle1)) # Expected Output: 0
    print("Converted Zigzag Output:", solution.strStr(haystack2, needle2)) # Expected Output: -1